.github/workflows/pull-request-checks.yaml

```diff
@@ -611,10 +611,7 @@ jobs:
           DEBIAN_FRONTEND: noninteractive
         run: |
           sudo apt-get update
-          sudo apt-get install --no-install-recommends -yq clang-format-11
-          wget -O git-clang-format https://raw.githubusercontent.com/llvm/llvm-project/llvmorg-14.0.6/clang/tools/clang-format/git-clang-format
-          chmod u+x git-clang-format
-          mv git-clang-format /usr/local/bin
+          sudo apt-get install --no-install-recommends -yq clang-format clang-format-11
```

Comment:
Why is this installing both `clang-format` and `clang-format-11`?

Mind you that both appear to install a `git-clang-format` binary, the first one installs `git-clang-format-10` and the second `git-clang-format-11`. Surely these clash in terms of path?

https://packages.ubuntu.com/focal-updates/amd64/clang-format-11/filelist

Reply:
The `clang-format` package installs `git-clang-format`, while the others install one with a version suffix. We need/want the one without a version suffix, hence the need to install both `clang-format` as well as `clang-format-11`.

Reply:
So the no suffix version of `git-clang-format` wouldn't be needed, if we just update `.github/workflows/pull-request-check-clang-format.sh` to run `git-clang-format-11` instead?

Reply:
I think that's true, yes. I'll give that one a try.

Reply:
Done.